.air.toml

```diff
@@ -0,0 +1,21 @@
+root = "."
+tmp_dir = ".temp"
```

Comment:
I just prefer `temp` name instead of tmp (This does not matter) and I have added `.` prefix to move `.temp` folder to the top of list items in vscode (This is easy to read for me)